pkgs/by-name/ju/jumper/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "v0"; # what should I put here? No version found anywhere
+  src = fetchFromGitHub {
+    owner = "homerours";
+    repo = "jumper";
+    rev = "264f291";
```

Comment:
Please use the full commit hash. Seven-character SHA1 strings are prone to collisions.